internal/distro/rhel8/azure.go

```diff
@@ -117,26 +118,12 @@ func azureEap7RhuiImgType() imageType {
 			},
 		},
 	}
-	it.defaultImageConfig.Firewall = &osbuild.FirewallStageOptions{
-		Ports: []string{
-			"23364:tcp", // modcluster
-			"25:tcp",    // mail
-			"3528:tcp",  // IIOP
-			"3529:tcp",  // IIOP-SSL
-			"45700:tcp", // Clustering subsystem
-			"4712:tcp",  // recovery manager
-			"4713:tcp",  // recovery manager
-			"54200:tcp", // Clustering subsystem
-			"54688:tcp", // Clustering subsystem
-			"55200:tcp", // Clustering subsystem
-			"57600:tcp", // Clustering subsystem
-			"7600:tcp",  // Clustering subsystem
-			"8009:tcp",  // AJP
-			"8080:tcp",  // HTTP
-			"8443:tcp",  // HTTPS
-			"9990:tcp",  // Management interface over HTTP
-			"9993:tcp",  // Management interface over HTTPS
-		},
+	it.defaultImageConfig.EnabledServices = []string{
+		"nm-cloud-setup.service",
+		"nm-cloud-setup.timer",
+		"sshd",
+		"systemd-resolved",
+		"waagent",
```

Comment:
2 nitpicks:

1. I wonder if it would not make sense to make this the default (as you did with the package set) and then just append the `firewalld` service in all other image types. Functionally it does not matter.
2. I would find it nicer to define the EAP specific image config on one line and then just inherit the rest of the defaults using `InheritFrom()` https://github.com/osbuild/osbuild-composer/blob/7224c896b69d668d039fe17ee2c5e7fc8f59c5e9/internal/distro/image_config.go#L74 method with the common Azure image config. I missed this initially when the EAP image type was added.

Reply:
Done!  Thanks.